src/__swaps__/screens/Swap/hooks/useSwapInputsController.ts

```diff
@@ -491,13 +491,12 @@ export function useSwapInputsController({
         return;
       }
 
+      const inputDecimals = inputAsset?.networks[inputAsset.chainId]?.decimals || inputAsset?.decimals;
+      const outputDecimals = outputAsset?.networks[outputAsset.chainId]?.decimals || outputAsset?.decimals;
```

Comment:
I think we should do 
```suggestion
      const inputDecimals = inputAsset?.networks[inputAsset.chainId]?.decimals ?? inputAsset?.decimals;
      const outputDecimals = outputAsset?.networks[outputAsset.chainId]?.decimals ?? outputAsset?.decimals;
```
maybe don't have to check if is a number using by `??`